src/styles/blocks/calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: left;
+  padding: 10px;
+  width: 7 * ($width-day + $gap-day);
+  gap: $gap-day;
+
+  .calendar__day {
+    display: flex;
+    box-sizing: border-box;
+    position: relative;
+    width: $width-day;
+    height: 100px;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+
+    border: 1px solid #000;
+    background-color: #eee;
+
+    transition: all 0.5s;
```

don't use it like that, because it's the same as transition: 0.5s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```